tests/flow/test_bfs.py

```
@@ -11,35 +8,22 @@ def __init__(self):
+        CREATE (a {v: 'a'})
+        CREATE (b {v: 'b'})
+        CREATE (c {v: 'c'})
+        CREATE (d {v: 'd'})
+        CREATE (e {v: 'e'})
+        CREATE (a)-[ab:E1 {v:'b'}]->(b)
+        CREATE (b)-[bc:E1 {v:'c'}]->(c)
+        CREATE (b)-[bd:E2 {v:'d'}]->(d)
+        CREATE (d)-[de:E1 {v:'e'}]->(e)
+        RETURN a, b, c, d, e, ab, bc, bd, de"""
+
+        res = self.graph.query(q).result_set
```

Ruff (0.11.9)

Local variable `res` is assigned to but never used

Remove assignment to unused variable `res`

(F841)